src/Microsoft.Health.Fhir.TemplateManagement/OCIFileManager.cs

```diff
@@ -28,10 +29,10 @@ public OCIFileManager(string imageReference, string workingFolder)
             _overlayOperator = new OverlayOperator();
         }
 
-        public async Task PullOCIImageAsync()
+        public async Task<string> PullOCIImageAsync()
```

Comment:
Seems PullOCIImageAsync is always followed by unpack? Can we make unpack a private function?